modules/ting_search/plugins/content_types/search_result_count.inc

```diff
@@ -22,25 +22,21 @@ function ting_search_result_count_content_type_render($subtype, $conf, $panel_ar
 
   $search_result = ting_search_current_results();
   if (isset($search_result)) {
-    $results = $search_result->getNumTotalObjects();
+    // OpenSearch only delivers the number of manifestions but the search 
+    // result can also include materials. Because of this, we need to display 
+    // different types of result counts based on the "more results" value,
+    // which basically checks if theres more than one page available for the 
+    // search result pagination.
+    $results = $search_result->hasMoreResults() ?
+      $search_result->getNumTotalObjects() :
+      $search_result->getNumTotalCollections();
```

Comment:
getNumTotalCollections() ændres til getNumCollections() jfr. https://platform.dandigbib.org/issues/3804

Reply:
Updated in d4a8b8b